src/main/kotlin/com/exactpro/th2/codec/EventProcessor.kt

```diff
@@ -152,10 +181,22 @@ class EventProcessor(
         }
     }
 
+    @Suppress("unused")
+    private class ReferenceToMessage(val messageId: String) : IBodyData {
+        val type: String
+            get() = TYPE
+
+        companion object {
+            const val TYPE = "reference"
```

Comment:
I don't believe that UI supports message ID in the body with type reference. And I am not sure that it will correctly handle this at all and this body won't break the UI

Reply:
Okay. But are you sure that this won't break the UI? If you are or you have checked already that those events can be displayed at least as a JSON in the UI we can continue with the current structure for referencing the message.

Reply:
I haven't checked but
* reverence to event has been using since 2022
* UI shows formatted JSON when content has unknown format

Reply:
> UI shows formatted JSON when content has unknown format

But the type is known in this case (as you said - `reverence to event has been using since 2022`). So, I believe we must check that in this case, the UI displays the content as JSON